frontend/src/views/components/Legend/Legend.component.jsx

```diff
@@ -13,6 +13,7 @@ const Legend = ({ activeLayers, reorder, loading, toggleLayer, setOpacity }) =>
   const [opened, toggleOpen] = useToggle(true);
   const onDragEnd = useCallback(
     ({ source, destination }) => destination && reorder(source.index, destination.index),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [],
   );
```

Comment:
```suggestion
  const onDragEnd = useCallback(
    ({ source, destination }) => !!destination && reorder(source.index, destination.index),
    [reorder],
  );
```
why not include `reorder` in the list of dependencies of the hook?